src/monitor-view-events.js

```diff
@@ -0,0 +1,53 @@
+// DOM Refresh
```

Comment:
i see renderer and templateCache as more of an object singeleton thing, but yea... shades of gray